demos/android/MASVS-RESILIENCE/MASTG-DEMO-0x41/output.txt

```diff
@@ -0,0 +1,39 @@
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.database.sqlite.SQLiteCursor.<init>(SQLiteCursor.java:98)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.database.sqlite.SQLiteDirectCursorDriver.query(SQLiteDirectCursorDriver.java:52)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.database.sqlite.SQLiteDatabase.rawQueryWithFactory(SQLiteDatabase.java:1823)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.database.sqlite.SQLiteDatabase.rawQuery(SQLiteDatabase.java:1755)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at com.example.strictmode2.MastgTest.triggerSqliteCursorLeak(MastgTest.kt:32)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at com.example.strictmode2.MastgTest.mastgTest(MastgTest.kt:11)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at com.example.strictmode2.MainActivity.onCreate(MainActivity.kt:20)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.Activity.performCreate(Activity.java:9002)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.Activity.performCreate(Activity.java:8980)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1526)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4030)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4235)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:112)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:174)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:109)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:81)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2636)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.os.Handler.dispatchMessage(Handler.java:107)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.os.Looper.loopOnce(Looper.java:232)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.os.Looper.loop(Looper.java:317)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at android.app.ActivityThread.main(ActivityThread.java:8705)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at java.lang.reflect.Method.invoke(Native Method)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:580)
+03-25 10:32:13.539  7952  7960 D StrictMode:    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:886)
```

Comment:
```suggestion
03-26 10:12:01.577  6218  6230 D StrictMode: StrictMode policy violation: android.os.strictmode.LeakedClosableViolation: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.
```

Reply:
I think the full stracktrace is more usable here, because it also shows _where_ the issue is. Otherwise, the log message is not that helpful for a reverse engineer.

Reply:
Makes sense!

Reply:
Please add something in the "Evaluation" section of the demo indicating where the issue is located as seen in the output.

Reply:
Updated suggestion, it includes the right package now: `org.owasp.mastestapp` instead of `com.example.strictmode2`

```sh
03-26 11:33:23.415  2340  2407 D StrictMode: 	at org.owasp.mastestapp.MastgTest.triggerSqliteCursorLeak(MastgTest.kt:35)
03-26 11:33:23.415  2340  2407 D StrictMode: 	at org.owasp.mastestapp.MastgTest.mastgTest(MastgTest.kt:12)
03-26 11:33:23.415  2340  2407 D StrictMode: 	at org.owasp.mastestapp.MainActivityKt$MyScreenContent$1$1$1.invoke(MainActivity.kt:117)
03-26 11:33:23.415  2340  2407 D StrictMode: 	at org.owasp.mastestapp.MainActivityKt$MyScreenContent$1$1$1.invoke(MainActivity.kt:115)
```